src/main/java/net/kodehawa/mantarobot/core/command/TextContext.java

```diff
@@ -496,6 +552,20 @@ public User getAuthor() {
         return getMessage().getAuthor();
     }
 
+    public Marriage getMarriage(MongoUser userData) {
+        return MantaroData.db().getMarriage(userData.getMarriageId());
+    }
```

Comment:
Both TextContext and BaseInteractionContext now have this method and both implementations have the exact same body. This should really be a default method in IContext.